### PR TITLE
Added CDS 4.0.0

### PR DESCRIPTION
**Pull request by n2iw**

### CODE REVIEW

File: CDS/4.0.0/cds-model-props.yml [hunk 1]
```diff
@@ -0,0 +1,12872 @@
+PropDefinitions:
+  #crdc_id
+  crdc_id:
+    Desc: The crdc_id is a unique identifier that is generated by Data Hub
+    Type: string
+    Tags:
+      Template: 'No
```
Comment (amattu2) [inline CDS/4.0.0/cds-model-props.yml:7]: Line 17 is missing a terminating `'` and is causing model parsing issues in Model Navigator.

**Review by amattu2 — CHANGES_REQUESTED** [review]
Syntax issue noted in the Model Props.

File: CDS/4.0.0/cds-model-props.yml (hunk 1 above)
Comment (n2iw) [reply CDS/4.0.0/cds-model-props.yml]: Updated.

**Review by amattu2 — APPROVED** [review]
Just mentioning this in case it's not on purpose:

- Treatment is now attached to a parent but still is marked as `Template: 'No'`
- We don't have any loading file examples for the Treatment node

Otherwise, it LGTM and loads in model navigator as expected.

**Pull request merged**